### ISSUE TITLE
Release Twisted 8.0.0 and all subprojects

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3085]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3085|
|Type|enhancement|
|Created|2008-03-17 15:50:08Z|
|Branch|https://github.com/twisted/twisted/tree/twisted-8.0.x-3085-4|

We need a release of Twisted!

<details><summary>Searchable metadata</summary>

```
trac-id__3085 3085
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__twisted_8_0 twisted-8.0
branch__branches_twisted_8_0_x_3085_4 branches/twisted-8.0.x-3085-4
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1205769008000000 1205769008000000
changetime__1206553780000000 1206553780000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix commented|
|-|-|

(In [22866]) Branching to 'twisted-8.0.x-3085'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix commented|
|-|-|

(In [22945]) Branching to 'twisted-8.0.x-3085-2'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix commented|
|-|-|

(In [23011]) Branching to 'twisted-8.0.x-3085-3'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix commented|
|-|-|

(In [23025]) Branching to 'twisted-8.0.x-3085-4'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_6"></a>|@radix removed owner|
|-|-|

The release is out. Need a review. Sorry, it's big!

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve set owner to @therve|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_8"></a>|@therve set owner to @radix|
|-|-|

 * INSTALL should mention Python 2.3 instead of 2.2
 * INSTALL mentions outdated debian repository
 * same thing for OS X
 * update copyright in README
 * no NEWS for web2
 * what about vfs ?

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_9"></a>|@radix commented|
|-|-|

Thanks, I've updated INSTALL and README. I wrote a NEWS for web2. I didn't write one for VFS, because there's never been a release nobody uses it.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @therve|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_11"></a>|@therve set owner to @radix|
|-|-|

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_12"></a>|@radix set status to `closed`|
|-|-|

(In [23101]) Merge twisted-8.0.x-3085-4

Author: radix
Reviewer: therve
Fixes [#3085](https://github.com/twisted/twisted/issues/3085)

This includes lots of release shrapnel from the release of Twisted 8.0.0
and 8.0.1. release notes were written, version numbers were updated,
and deprecation warnings were updated.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
